tdrs-frontend/nginx/nginx.conf

```diff
@@ -2,5 +2,5 @@ server {
     listen 80;
     root /usr/share/nginx/html;
     include locations.conf;
-
+    add_header Content-Security-Policy "default-src 'none'; script-src 'self'; style-src 'self' 'unsafe-inline'; img-src 'self' data:; font-src 'self'; connect-src 'self' *.cloud.gov; manifest-src 'self'; object-src 'none'; frame-ancestors 'none'; form-action 'none';";
```

Comment:
Does this still work without `unsafe-inline` since the `INLINE_RUNTIME_CHUNK` is set to false below?

Additionally, this particular nginx configuration is used by Circle CI - so it won't be hosted on `*.cloud.gov`, although I'm not sure if that is an issue or not.

Reply:
It does not work without the `unsafe-inline` even when `INLINE_RUNTIME_CHUNK=false`.

Reply:
Gotcha, seems fitting to split that off as a separate issue. I think you're right in that its coming from either `node-sass` or `uswds` because that should work with an un-ejected create-react-app.